samples/maps/series/mappoint-animation/demo.js

```diff
@@ -0,0 +1,84 @@
-76,8],[-42,18],[39,19],[-28,32],[84,6],[26,29],[58,8],[71,-32],[67,-87],[81,0],[106,-31],[56,-57],[-3,-25],[-67,0],[-37,-25],[73,2]],[[7610,9580],[65,52],[20,-11],[-59,-39],[42,5],[-9,-38],[-76,-7],[80,-25],[-51,-27],[39,-17],[-61,-19],[-28,-44],[45,-15],[-7,-35],[-49,-44],[39,-38],[-45,-18],[22,-39],[-46,-1],[-21,-105],[-84,24],[45,44],[-22,19],[38,47],[33,79],[-42,61],[38,59],[-73,-44],[-34,12],[-47,-50],[-59,34],[50,23],[-30,23],[-92,-25],[-45,22],[-17,42],[27,19],[145,10],[11,-35],[18,38],[66,0],[30,30],[55,2],[-110,32],[8,30],[-44,7],[-6,57],[-133,-8],[79,68],[20,-30],[64,51],[-7,25],[75,3],[21,30],[7,-41],[-35,-61],[0,-43],[33,6],[-61,-83],[37,23],[74,16],[-21,19],[35,30],[53,-61],[0,-39]],[[7609,9690],[-22,107],[6,19],[57,-28],[-17,41],[9,51],[28,19],[70,-4],[0,-89],[-57,26],[39,-44],[-31,-16],[34,-19],[-24,-22],[7,-48],[-99,7]],[[2541,6690],[66,-24],[-4,-63],[50,-31],[-20,-13],[-91,0],[-32,27],[-19,79],[-34,19],[53,70],[-31,70],[62,18],[73,-32],[11,-38],[-99,52],[0,-31],[104,-27],[35,-31],[-46,-51],[-78,6]],[[2680,6976],[40,-14],[-19,-26],[-106,-2],[-53,41],[-75,24],[-17,-12],[-67,25],[56,64],[74,-1],[-8,-25],[52,26],[46,-1],[79,25],[-6,-31],[78,-3],[23,-16],[-28,-25],[-60,32],[-37,-1],[55,-72],[-66,-9],[-22,18],[-44,-26],[105,9]],[[3448,7590],[67,34],[0,-39],[-44,-25],[-151,18],[22,-63],[-95,-38],[-29,7],[-77,-26],[184,21],[33,-58],[-58,-31],[-98,12],[73,-38],[-68,-51],[-93,3],[-47,74],[35,37],[-18,12],[-62,-50],[73,-68],[12,-45],[-96,20],[-23,-20],[56,-37],[-84,-25],[-43,-42],[-73,-30],[-97,71],[113,53],[1,23],[89,15],[10,22],[-161,23],[-61,27],[44,32],[100,26],[-101,-13],[29,38],[-42,-16],[-41,57],[10,52],[78,8],[-30,11],[19,39],[100,-25],[-24,-5],[41,-39],[73,31],[50,-32],[23,20],[-61,38],[-28,50],[84,49],[77,11],[225,103],[64,45],[51,-28],[31,-57],[-38,-47],[38,-20],[-67,-31],[-3,-18],[-93,-51],[46,-21],[55,7]],[[4847,4048],[74,35],[74,9],[-21,-48],[11,-35],[41,-33],[-63,-86],[-70,-49],[-95,-46],[-17,-31],[-74,26],[-71,-20],[43,56],[22,88],[92,51],[54,83]],[[2627,4694],[149,-24],[40,25],[34,60],[10,54],[65,-19],[105,4],[93,28],[57,2],[78,34],[48,-15],[32,13],[100,-32],[78,10],[48,-9],[52,-41],[-20,-86],[71,-19],[-10,-66],[47,-10],[82,-79],[23,-58],[55,-14],[-52,35],[38,11],[40,-29],[7,-54],[-29,-34],[-94,-26],[-67,-62],[25,-25],[46,40],[75,28],[44,-10],[97,10],[31,-20],[19,-55],[29,-26],[8,-49],[28,-46],[-21,-68],[-34,-48],[-61,58],[24,78],[-24,38],[-55,35],[-47,9],[-12,-37],[74,-19],[0,-69],[-29,-56],[-39,-26],[106,29],[23,-17],[-2,-61],[-83,-64],[-19,16],[-83,0],[-68,-14],[-4,-64],[-37,-65],[-104,-50],[-109,35],[-10,17],[-69,5]]],"bbox":[-10.461181018243009,49.95912154352611,1.77056437025751,60.845036413947454],"transform":{"scale":[0.0012232968685369056,0.0010887003570778423],"translate":[-10.461181018243009,49.95912154352611]},"title":"British Isles","version":"2.0.1","copyright":"Copyright (c) 2022 Highsoft AS, Based on data from Natural Earth","copyrightShort":"Natural Earth","copyrightUrl":"http://www.naturalearthdata.com"}';
```

Comment:
Any reason why we are not just loading the TopoJSON from our maps server? Like other demos:

https://github.com/highcharts/highcharts/blob/fa98f7ebc8d618179c4337a03356712b42a78f20/samples/maps/accessibility/accessible-data-classes/demo.js#L3-L5

Reply:
Right, it wasn't necessary for the manual test.